### PR TITLE
Prepare v1

### LINKED ISSUE #77: Change the wording for the widget fields

### PR DESCRIPTION
**Pull request by almet**
Fixes #77 #76 #72 #71 #70 #69 
This merges opened pull requests #65 #78 and #79 

I've marked some tests as skipped but this wasn't introduced by this very pull request.

### CODE REVIEW

**Comment by almet** [comment]
I would love to understand why I have to skip these tests. It appears that it's related to the `Simulate.change` and `Simulate.submit` operations. I wasn't able to find out what's happening.

File: devServer.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var path = require("path");
 var express = require("express");
 var webpack = require("webpack");
 
-var env = "dev", port = 8080;
+var env = "dev", port = 9000;
```
Comment (Natim) [inline devServer.js]: nit: Is it intended to change the default port for the formbuilder?

**Comment by Natim** [comment]
It seems that the description edit doesn't work.

**Comment by Natim** [comment]
I have deployed the branch on http://kinto.github.io/formbuilder/

**Comment by almet** [comment]
I've redeployed it and it's now possible to edit the description.

**Comment by Natim** [comment]
LGTM :)

**Comment by almet** [comment]
I've redeployed with a different theme, I think it's better. What do you think?

**Comment by Natim** [comment]
Except for the Reset form text color it looks better yes.

**Pull request merged**